docs/guide/displaying_children_of_a_content_item.md

```diff
@@ -3,12 +3,13 @@
 One of the basic design tasks you may need to complete when creating your website is configuring one page to display all of its children.
 For example you can configure a blog displaying all blog posts or a folder showing all articles it contains.
 
-There are two ways to make a Content item display its children:
+There are three ways to make a Content item display its children:
 
 1. [Using the Query Controller](#using-the-query-controller)
+1. [Using the Content query Field Type](#using-the-content-query-field-type)
```

Comment:
I think that the query field should be the first in the list, as it is the easiest way to achieve the task. The query controller should be second, and last should be the custom controller.